client/src/pages/NoPageFound/NoPageFound.test.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import { BrowserRouter } from 'react-router-dom';
+
+import NoPageFound from '.';
```

Comment:
maybe keep it consistent with the text that it renders, such as: PageNotFound.jsx?

Reply:
Sounds good, will make that change